### ISSUE TITLE
Fix the namespace used for the "modelSpecificMetadata" and its children metadata

### ISSUE THREAD

**Issue by imanmaghami** [hydroshare/hydroshare#4269]
The newly added term for the model instance aggregation type (i.e., modelSpecificMetadata) is not a hsterm in a meta.xml file for a model instance aggregation but rather a different namespace (i.e., xmlns:default# = 'resource url', where default# is the name of the namespace, # is a number and resource url is the url for the HydroShare resource url that hosts the model aggregation) is considered for it and its children metadata. For the "modelSpecificMetadata" namespace hsterms should be used while **no** namespace should be used for its children's metadata.

**Additionally,** there is a problem in model instance meta.xml for the following specific case: 
If a model instance that has schema based metadata fields filled out by the user, gets bag downloaded, and then the schema associated with the model program that the model instance is linked to get changed in a way that the old schema based metadata is invalid against the new schema, after bag downloading the model instance, the model instance meta.xml file surprisingly shows two namespaces default#X and default#Y (where #X and #Y are integer numbers) where both point to the same HydroShare resource that contains the model aggregation files. Please note that this behavior was experienced ONLY if the model instance gets bag download before making the change in the schema making the old metadata invalid against the edited schema.

In short, in the following screenshot, make the following changes:
Line 3 - remove default16
Line 5 remove
Line 11 replace default16 by hsterms
Line 13 remove default16

![image](https://user-images.githubusercontent.com/33031286/121797955-74ad3f80-cbf1-11eb-962a-52a2ce2293e1.png)

**Comment by pkdash** [comment]
@imanmaghami Locally I could not produce this error:

**Comment by eclark-cuahsi** [comment]
Adding @horsburgh and @goodalljl as this falls under the Resource Model and Model Aggregation category.
